docs/pages/versions/unversioned/sdk/webbrowser.md

```diff
@@ -36,6 +36,15 @@ Opens the url with Safari in a modal on iOS using `SFSafariViewController`, and
 #### Arguments
 
 - **url (_string_)** -- The url to open in the web browser.
+- **arguments (_object_)** (_optional_) --
+  A dictionaty with following key-value pairs:
+
+  - **toolbarColor (_optional_) (_string_)** : _Android only_. color of toolbar to be used by Custom Tab on Android. Required format `#AARRGGBB` or `#RRGGBB`. This is supported only on Android.
+  - **enableBarCollapsing (_optional_) (_boolean_)** : flag determinig whether toolbar might be hiding when user scrolls website.
+  - **showTitle (_optional_) (_boolean_)** : _Android only_. Flag determining whether browser should show title of website on Toolbar.
+  - **package (_optional_) (_string_)** : _Android only_. Package name of a browser to be used to handle Custom Tabs. List of available packages is to be queried by [getCustomTabsSupportingBrowsers](#WebBrowser.getCustomTabsSupportingBrowsers) method.
```

Comment:
```suggestion
  - **browserPackage (_optional_) (_string_)** : (_Android only_) package name of the browser to be used when opening the URL. List of available packages is to be queried by [getCustomTabsSupportingBrowsers](#WebBrowser.getCustomTabsSupportingBrowsers) method
```

+ please run the docs server and check what will be the real anchor tag, it won't contain `.`, that's for sure

Reply:
Will change package to browserPackage in next PR if that's okay. I just want to minimize conflicts.